builder/ebssurrogate/step_swap_volumes.go

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package ebssurrogate
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	awscommon "github.com/hashicorp/packer-plugin-amazon/builder/common"
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+	"github.com/hashicorp/packer-plugin-sdk/template/interpolate"
+)
+
+// StepSnapshotVolumes creates snapshots of the created volumes.
```

Comment:
This looks to be the comment from SnapshotVolumes

```suggestion
// StepSnapshotVolumes creates snapshots of the created volumes.
```